soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -166,6 +166,15 @@ object SoQLFunctions {
   val Overlaps = f("overlaps", FunctionName("overlaps"), Map("a" -> GeospatialLike, "b" -> GeospatialLike),
     Seq(VariableType("a"), VariableType("b")), Seq.empty, FixedType(SoQLBoolean))(
     "Return the rows that where the locations 'spatially overlap', meaning they intersect, but one does not completely contain another and they share interior points")
+  
+  val Intersection = f("intersection", FunctionName("intersection"), Map("a" -> GeospatialLike, "b" -> GeospatialLike),
+    Seq(VariableType("a"), VariableType("b")), Seq.empty, VariableType("a"))(
+    "Returns the geometry of the intersection between two geometries")
+  
+  val Area = f("area", FunctionName("area"), Map("a" -> GeospatialLike),
+    Seq(VariableType("a")), Seq.empty, FixedType(SoQLNumber))(
+    "Returns the area of the geometry")
```

Comment:
..is this useful on non-polygon-like geometry types?

Reply:
technically possible, but less than useful - updated to make them specifically poly/multipolygon types